apps/laboratory/src/pages/api/auth/[...nextauth].ts

```diff
@@ -47,22 +45,19 @@ export default async function auth(req: NextApiRequest, res: NextApiResponse) {
           if (!credentials?.message) {
             throw new Error('SiweMessage is undefined')
           }
-          const siwe = new SiweMessage(credentials.message)
-          const provider = new ethers.JsonRpcProvider(
-            `https://rpc.walletconnect.com/v1?chainId=eip155:${siwe.chainId}&projectId=${projectId}`
-          )
-          const nonce = await getCsrfToken({ req: { headers: req.headers } })
-          const result = await siwe.verify(
-            {
-              signature: credentials?.signature || '',
-              nonce
-            },
-            { provider }
-          )
+          const { message, signature } = credentials
+          const address = message.match(/0x[a-fA-F0-9]{40}/u)?.[0] || ''
```

Comment:
I looks to me that these kind of regex values should be stored in a constant with a clear name so it could improve the readability and maintainability. 

There could be a map like the following:
```js
const regexPatterns = {
   address: /0x[a-fA-F0-9]{40}/u,
   chainId: /Chain ID: (?<temp1>\d+)/u,
   ...
}
```